### PR TITLE
add kustomize manifest and update script

### PR DESCRIPTION
**Pull request by guangbochen**
**Problem:**
Add rancher-vm `all-in-one` YAML file for quick deployment test and validation.
Notes: this may seem duplicate of the vm helm chart, but would be nice to have an all-in-one file for non-helm users.

**Solution:**
1. add VM kustomize manifest and CI integration script

**Related Issue/PR:**
N/A

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -36,6 +37,29 @@ steps:
...
+      from_secret: docker_password
+    repo: "rancher/vm"
+    tag: ${DRONE_BRANCH}-head-amd64
+    username:
+      from_secret: docker_username
+  when:
+    ref:
+      include:
+      - "refs/heads/master"
+      - "refs/heads/release/v*"
```
Comment (orangedeng) [inline .drone.yml:59]: Do we really need to push master after tag?

File: .drone.yml [hunk 2]
```diff
@@ -89,6 +113,7 @@ steps:
     checksum_flatten: true
     files:
     - "dist/artifacts/*"
+    - "deploy/e2e/*"
```
Comment (orangedeng) [inline .drone.yml:116]: What kind of files we need to push to github in e2e*

**Review by orangedeng — COMMENTED** [review]
What's the point for us to build multiple arch images? I don't think kubevirt is supported in arm/arm64 platform.

**Comment by guangbochen** [comment]
@orangedeng since we already have the helm chart so we will use it instead of the kustomize generator, close this PR for now, thanks.

**Pull request closed without merging**